scala-package/spark/src/test/scala/org/apache/mxnet/spark/SharedSparkContext.scala

```diff
@@ -81,17 +81,15 @@ trait SharedSparkContext extends FunSuite with BeforeAndAfterEach with BeforeAnd
   }
 
   private def getJarFilePath(root: String): String = {
-    for (platform <- List("linux-x86_64-cpu", "linux-x86_64-gpu", "osx-x86_64-cpu")) {
```

Comment:
by removing this for loop; does it mean the path is independent of platform?

Reply:
Yes. The path for the jars is platform agnostic now.